packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/NodeComponent.java

```diff
@@ -187,7 +193,11 @@ private Element doRender(VariableManager nodeVariableManager, String targetObjec
                 .orElse(Position.UNDEFINED);
 
         Size size = this.getSize(optionalPreviousNode, nodeDescription, nodeVariableManager);
-        Set<CustomizableProperties> customizableProperties = optionalPreviousNode.map(Node::getCustomizedProperties).orElse(Set.of());
+
+        Set<CustomizableProperties> customizableProperties = Set.of();
+        if (CollapsingState.EXPANDED.equals(collapsingState)) {
```

Comment:
Does that mean that we will never keep the size of a manually resized collapsed node? Because I would like to resize my collapsed nodes 😛

Reply:
We will need to discuss about that, I'll just open an issue for now to track this